app/jobs/identities_backfill_job.rb

```diff
@@ -0,0 +1,54 @@
+class IdentitiesBackfillJob < ApplicationJob
+  # This is a short-term solution to backfill data requiring a table scan.
+  # This job can be deleted once it's done.
+
+  queue_as :low
+
+  # Let's give us the option to fine-tune this on the fly
+  BATCH_SIZE_KEY = 'IdentitiesBackfillJob.batch_size'.freeze
+  SLICE_SIZE_KEY = 'IdentitiesBackfillJob.slice_size'.freeze
+  CACHE_KEY = 'IdentitiesBackfillJob.position'.freeze
+
+  def perform
+    start_time = Time.now
+    max_id = ServiceProviderIdentity.last.id
+
+    (batch_size / slice_size).times.each do |slice_num|
+      start_id = position + (slice_size * slice_num)
+      next if start_id > max_id
+      sp_query = <<~SQL
+UPDATE identities
+SET last_consented_at = created_at
+WHERE id > #{start_id}
+AND id <= #{start_id + slice_size}
+AND deleted_at IS NULL
+AND last_consented_at IS NULL
+      SQL
```

Comment:
1. the `~` in `<<~` means leading spaces are trimmed so we can indent this
2. I think it's a good habit to escape params in raw SQL, even if here they're integers

```suggestion
      params = {
        min_id: start_id,
        max_id: start_id + slice_size
      }.transform_values { |v| ActiveRecord::Base.connection.quote(v) }
      sp_query = format(<<~SQL, params)
        UPDATE identities
        SET last_consented_at = created_at
        WHERE id > %{min_id}
        AND id <= %{max_id}
        AND deleted_at IS NULL
        AND last_consented_at IS NULL
      SQL
```